### PR TITLE
Traducao/json routing and embedding

### PR DESCRIPTION
**Pull request by Ronald-TR**
Ainda falta revisar, ajustar bastante coisa e traduzir os códigos, mas gostaria de deixar o PR aberto.

Ps: Desculpem qualquer coisa.
Abraços!!

### CODE REVIEW

File: criando-uma-aplicacao/json.md [hunk 1]
```diff
@@ -676,17 +670,19 @@ func assertContentType(t *testing.T, response *httptest.ResponseRecorder, want s
...
+
+A forma mais rápida de nós termos alguma confiança é adicionar a nosso teste de integração, nós podemos bater no novo endpoint e checar se nós recebemos a resposta correta de `/league`.
 
-The quickest way for us to get some confidence is to add to our integration test, we can hit the new endpoint and check we get back the correct response from `/league`.
+## Escreva o teste primeiro
 
-## Write the test first
+Nós podemos usar `t.Run` para parar este teste um pouco e então reusar os helpers dos testes do nosso servidor - novamente mostrando a importancia de refatoração dos testes.
 
 We can use `t.Run` to break up this test a bit and we can reuse the helpers from our server tests - again showing the importance of refactoring tests.
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:687]: Trecho em en_US que foi traduzido e tá sobrando aqui :slightly_smiling_face:

File: criando-uma-aplicacao/json.md [hunk 2]
```diff
@@ -744,17 +740,17 @@ func (i *InMemoryPlayerStore) GetLeague() []Player {
 }
 ```
 
-All we need to do is iterate over the map and convert each key/value to a `Player`.
+Tudo que nós precisamos fazer é interar através do map e converter cada chave/valor para um `Jogador`
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:743]: `iterar` ao invés de `interar` :wink:

File: criando-uma-aplicacao/json.md [hunk 3]
```diff
@@ -744,17 +740,17 @@ func (i *InMemoryPlayerStore) GetLeague() []Player {
...
 
-The test should now pass.
+O teste deve passar agora.
 
-## Wrapping up
+## Enpacotando
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:747]: O `Wrapping up` aqui tem um significado de "resumão" de tudo o que foi visto até aqui, então, que tal se mudarmos a tradução para `Concluindo`? :wink:

File: criando-uma-aplicacao/json.md [hunk 4]
```diff
@@ -744,17 +740,17 @@ func (i *InMemoryPlayerStore) GetLeague() []Player {
...
 
-We've continued to safely iterate on our program using TDD, making it support new endpoints in a maintainable way with a router and it can now return JSON for our consumers. In the next chapter, we will cover persisting the data and sorting our league.
+Nós temos continuado a seguramente iterar no nosso programa usando TDD, fazendo ele suportar novos endpoints de uma forma manutenivel com um roteador e isso pode agora retornar JSON para nossos consumidores. No próximo capítulo, nós vamos cobrir persistência de dados e ordenação de nossas ligas.
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:749]: `manutenível` ao invés de `manutenivel` :wink:

File: criando-uma-aplicacao/json.md [hunk 5]
```diff
@@ -744,17 +740,17 @@ func (i *InMemoryPlayerStore) GetLeague() []Player {
...
-* **JSON deserializing and serializing**. The standard library makes it very trivial to serialise and deserialise your data. It is also open to configuration and you can customise how these data transformations work if necessary.
+* **Roteamento**. A biblioteca padrão oferece uma fácil forma de usar tipos para fazer roteamento. Ela abraça completamente a interface `http.Handler` nela, tanto que você pode atribuir rotas para `Handler`s e a rota em si também é um `Handler`. Ela não tem alguns recursos que você pode esperar, como caminhos para variáveis \(ex. `/users/{id}`\). Você pode facilmente analisar esta informação por si mesmo porém você pode querer considerar olhar para outras bibliotecas de roteamento se isso se tornar um fardo. Muitas das mais populares seguem a filosofia das bibliotecas padrões e também implementam `http.Handler`.
 
+* **Composição**. Nós tocamos um pouco nesta tecnica porém você pode [ler mais sobre isso de Effective Go](https://golang.org/doc/effective_go.html#embedding). Se há uma coisa que você deve tirar disso é que composições podem ser extremamente úteis, porém _sempre pensando na sua API pública, só exponha o que é apropriado_.
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:755]: `técnica` ao invés de `tecnica` :wink:

File: criando-uma-aplicacao/json.md [hunk 6]
```diff
@@ -744,17 +740,17 @@ func (i *InMemoryPlayerStore) GetLeague() []Player {
...
+* **JSON desserialização e serialização**. A biblioteca padrão faz isto de forma bastante trivial ao serializar e desserializar nosso dado. Isto também abre para configurações e você pode customizar como esta transformação de dados funciona se necessário.
```
Comment (devdrops) [inline criando-uma-aplicacao/json.md:756]: `Serialização e Desserialização de JSON` ao invés de `JSON desserialização e serialização` :wink:

**Comment by devdrops** [comment]
Fala @Ronald-TR! Tudo certo?

Precisa de alguma ajuda com este PR?

**Comment by devdrops** [comment]
Salve @larien!

Precisa de ajuda neste PR?

File: criando-uma-aplicacao/json.md (hunk 6 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md (hunk 5 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md (hunk 4 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md (hunk 3 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md (hunk 2 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 7]
```diff
@@ -134,19 +135,19 @@ github.com/quii/learn-go-with-tests/json-and-io/v2.(*PlayerServer).ServeHTTP(0xc
...
 ```
 
-In the previous chapter, we mentioned this was a fairly naive way of doing our routing. What is happening is it's trying to split the string of the path starting at an index beyond `/league` so it is `slice bounds out of range`.
+No capítulo anterior, nós mencionamos que isto era uma maneira bastante ingênua de fazer o nosso roteamento. O que está acontecendo é que ele está tentando cortar a string do caminho da URL começando do índice após `/league` e então, isto nos dá um `slice bounds out of range`.
 
-## Write enough code to make it pass
+## Escreva somente o código suficiente para fazê-lo passar
 
-Go has a built-in routing mechanism called [`ServeMux`](https://golang.org/pkg/net/http/#ServeMux) \(request multiplexer\) which lets you attach `http.Handler`s to particular request paths.
+Go têm um mecanismo de rotas nativo (built-in) chamado [`ServeMux`](https://golang.org/pkg/net/http/#ServeMux) \(request multiplexer\) que nos permite atracar um `http.Handler`s para caminhos de uma requisição em específico.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 8]
```diff
@@ -115,10 +117,9 @@ func TestLeague(t *testing.T) {
     })
 }
 ```
+Antes de nos preocuparmos sobre as pontuações atuais e o JSON, nós vamos tentar manter as mudanças pequenas com o plano de ir passo a passo rumo ao nosso objetivo. O inicio mais simples é checar se nós conseguimos consultar `/league` e obter um `OK` de retorno.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 9]
```diff
@@ -1,12 +1,12 @@
 # JSON, roteamento and embedding
 
-[**You can find all the code for this chapter here**](https://github.com/quii/learn-go-with-tests/tree/master/json)
+[**Você pode encontrar todo o código para este capítulo aqui**](https://github.com/quii/learn-go-with-tests/tree/master/json)
 
-[In the previous chapter](http-server.md) we created a web server to store how many games players have won.
+[No capítulo anterior](http-server.md) nós criamos um web server para armazenar quantos jogos nossos jogadores venceram.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 10]
```diff
@@ -1,12 +1,12 @@
 # JSON, roteamento and embedding
 
-[**You can find all the code for this chapter here**](https://github.com/quii/learn-go-with-tests/tree/master/json)
+[**Você pode encontrar todo o código para este capítulo aqui**](https://github.com/quii/learn-go-with-tests/tree/master/json)
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 11]
```diff
@@ -1,12 +1,12 @@
 # JSON, roteamento and embedding
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 12]
```diff
@@ -172,16 +173,14 @@ func (p *PlayerServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 ```
 
-* When the request starts we create a router and then we tell it for `x` path use `y` handler.
-* So for our new endpoint, we use `http.HandlerFunc` and an _anonymous function_ to `w.WriteHeader(http.StatusOK)` when `/league` is requested to make our new test pass.
-* For the `/players/` route we just cut and paste our code into another `http.HandlerFunc`.
-* Finally, we handle the request that came in by calling our new router's `ServeHTTP` \(notice how `ServeMux` is _also_ an `http.Handler`?\)
+* Quando a requisição começa nós criamos um router e então dizemos para o caminho `x` usar o handler `y`.
+* Então para nosso novo endpoint, nós usamos `http.HandlerFunc` e uma _função anônima_ para `w.WriteHeader(http.StatusOK)` quando `/league` é requisitada para fazer nosso novo teste passar.
+* Para a rota `/players/` nós somente recortamos e colamos nosso codigo dentro de outro `http.HandlerFunc`.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 13]
```diff
@@ -209,7 +208,7 @@ func (p *PlayerServer) playersHandler(w http.ResponseWriter, r *http.Request) {
 }
 ```
 
-It's quite odd \(and inefficient\) to be setting up a router as a request comes in and then calling it. What we ideally want to do is have some kind of `NewPlayerServer` function which will take our dependencies and do the one-time setup of creating the router. Each request can then just use that one instance of the router.
+É um pouco estranho \(e ineficiente\) estar configurando um router quando uma requisição chegar e então chama-lo. O que idealmente queremos fazer é uma função do tipo `NewPlayerServer` que pegará nossas dependências e ao ser chamada, irá fazer a configuração única da criação do router. Desta forma, cada requisição pode usar somente uma instância do nosso router.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 14]
```diff
@@ -263,23 +262,25 @@ func NewPlayerServer(store PlayerStore) *PlayerServer {
 }
 ```
 
-Finally make sure you **delete** `func (p *PlayerServer) ServeHTTP(w http.ResponseWriter, r *http.Request)` as it is no longer needed!
+finalmente, se certifique de que você **deletou** `func (p *PlayerServer) ServeHTTP(w http.ResponseWriter, r *http.Request)` por não ser mais necessária!
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 15]
```diff
@@ -263,23 +262,25 @@ func NewPlayerServer(store PlayerStore) *PlayerServer {
...
+
+## Incorporando
 
-## Embedding
+Nós mudamos a segunda propriedade de `PlayerServer` removendo a propriedade nomeada `router http.ServeMux` e substituindo por `http.Handler`; isto é chamado de _incorporar_. 
 
-We changed the second property of `PlayerServer`, removing the named property `router http.ServeMux` and replaced it with `http.Handler`; this is called _embedding_.
 
+        todo, fiquei sem saber como traduzir este trecho abaixo:
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Obrigado pela sugestão, a adotei haha!

File: criando-uma-aplicacao/json.md [hunk 16]
```diff
@@ -263,23 +262,25 @@ func NewPlayerServer(store PlayerStore) *PlayerServer {
...
 > Go does not provide the typical, type-driven notion of subclassing, but it does have the ability to “borrow” pieces of an implementation by embedding types within a struct or interface.
 
 [Effective Go - Embedding](https://golang.org/doc/effective_go.html#embedding)
 
-What this means is that our `PlayerServer` now has all the methods that `http.Handler` has, which is just `ServeHTTP`.
+O que isto quer dizer é que nosso `PlayerServer` agora tem todos os métodos que `http.Handler` têm, que é somente o `ServeHTTP`.
 
-To "fill in" the `http.Handler` we assign it to the `router` we create in `NewPlayerServer`. We can do this because `http.ServeMux` has the method `ServeHTTP`.
+Para "preencher" o `http.Handler` nós atribuimos ele para o `router` que nós criamos em `NewPlayerServer`. Nós podemos fazer isso porque `http.ServeMux` tem o método `ServeHTTP`.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 17]
```diff
@@ -288,21 +289,21 @@ type Animal interface {
...
+**Quando incorporamos tipos, realmente devemos pensar sobre qual o impacto que isto terá em nossa API pública**
 
-It is a _very_ common mistake to misuse embedding and end up polluting your APIs and exposing the internals of your type.
+Isto é um erro _muito_ comum de mau uso de incorporamento, que termina poluindo nossas APIs e expondo os métodos internos dos seus tipos incorporados.
 
-Now we've restructured our application we can easily add new routes and have the start of the `/league` endpoint. We now need to make it return some useful information.
+Agora que nós reestruturamos nossa aplicação, nós podemos facilmente adicionar novas rotas e botar para funcionar nosso endpoint `/league`. Agora precisamos fazê-lo retornar algumas informações úteis.
 
-We should return some JSON that looks something like this.
+Nós poderíamos retornar um JSON semelhante a este:
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 18]
```diff
@@ -317,9 +318,8 @@ We should return some JSON that looks something like this.
 ]
 ```
 
-## Write the test first
-
-We'll start by trying to parse the response into something meaningful.
+## Escreva o teste primeiro
+Nós vamos começar tentando analizar a resposta dentro de algo mais significativo.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 19]
```diff
@@ -344,55 +344,55 @@ func TestLeague(t *testing.T) {
...
-var got []Player
+var got []Jogador
 err := json.NewDecoder(response.Body).Decode(&got)
 ```
 
-To parse JSON into our data model we create a `Decoder` from `encoding/json` package and then call its `Decode` method. To create a `Decoder` it needs an `io.Reader` to read from which in our case is our response spy's `Body`.
+Para analizar o JSON dentro de nosso modelo de dados nós criamos um `Decoder` do pacote `encoding/json` e então chamamos seu método `Decode`. Para criar um `Decoder` é necessário ler de um `io.Reader`, que em nosso caso é nossa própria resposta `Body`.
 
-`Decode` takes the address of the thing we are trying to decode into which is why we declare an empty slice of `Player` the line before.
+`Decode` pega o endereço da coisa que nós estamos tentando decodificar, e é por isso que nós declaramos uma fatia vazia de `Jogador` na linha anterior.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 20]
```diff
@@ -344,55 +344,55 @@ func TestLeague(t *testing.T) {
...
 
-Parsing JSON can fail so `Decode` can return an `error`. There's no point continuing the test if that fails so we check for the error and stop the test with `t.Fatalf` if it happens. Notice that we print the response body along with the error as it's important for someone running the test to see what string cannot be parsed.
+Esse processo de analisar um JSON pode falhar, então `Decode` pode retornar um `error`. Não há ponto de continuidade para o teste se isto acontecer, então nós checamos o erro e paramos o teste com `t.Fatalf`.
+Note que nós exibimos que o corpo da resposta junto com o erro, pois é importante para qualquer outra pessoa que esteja rodando os testes ver que o texto não pôde ser analisado.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 21]
```diff
@@ -434,13 +432,13 @@ func (p *PlayerServer) getLeagueTable() []Player{
...
+Mais adiante, nós vamos querer estender nossos testes para então podermos controlar exatamente qual dado nós queremos receber de volta.
 
-## Write the test first
+## Escreva o teste primeiro
 
-We can update the test to assert that the league table contains some players that we will stub in our store.
+Nós podemos atualizar o teste para afirmar que a tabela das ligas contem alguns jogadores que nós vamos pôr em nossa loja.
 
-Update `StubPlayerStore` to let it store a league, which is just a slice of `Player`. We'll store our expected data in there.
+Atualize `StubPlayerStore` para permitir que ele armazene uma liga, que é apenas uma fatia de `Jogador`. Nós vamos armazenar nossos dados esperados lá.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 22]
```diff
@@ -487,28 +484,27 @@ func TestLeague(t *testing.T) {
...
 ```
+## Escreva o minimo de código para que o teste rode e cheque as falhas na saída dele.
 
-## Write the minimal amount of code for the test to run and check the failing test output
-
-You'll need to update the other tests as we have a new field in `StubPlayerStore`; set it to nil for the other tests.
+Você vai precisar atualizar os outros testes, assim como nós temos um novo campo em `StubPlayerStore`; ponha-o como nulo para os outros testes.
 
-Try running the tests again and you should get
+Tente rodando os testes novamente e você deverá ter:
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 23]
```diff
@@ -487,28 +484,27 @@ func TestLeague(t *testing.T) {
...
 
 ```text
 === RUN   TestLeague/it_returns_the_league_table_as_JSON
     --- FAIL: TestLeague/it_returns_the_league_table_as_JSON (0.00s)
         server_test.go:124: got [{Chris 20}] want [{Cleo 32} {Chris 20} {Tiest 14}]
 ```
 
-## Write enough code to make it pass
+## Escreva código suficiente para faze-lo passar
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 24]
```diff
@@ -527,7 +523,7 @@ func (p *PlayerServer) leagueHandler(w http.ResponseWriter, r *http.Request) {
 }
 ```
 
-Try and run the tests.
+Tente e rode os testes:
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 25]
```diff
@@ -543,41 +539,39 @@ Try and run the tests.
...
 func (i *InMemoryPlayerStore) GetLeague() []Player {
     return nil
 }
 ```
 
-What this is really telling us is that _later_ we're going to want to test this but let's park that for now.
+O que isto está realmente nos dizendo é que _depois_ nós vamos querer testar isto, porém vamos estacionar isto por hora.
 
-Try and run the tests, the compiler should pass and the tests should be passing!
+Tente e rode os testes, o compilador deve passar e os testes deverão estar passando!
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 26]
```diff
@@ -543,41 +539,39 @@ Try and run the tests.
...
 
-## Refactor
+## Refatoração
 
-The test code does not convey out intent very well and has a lot of boilerplate we can refactor away.
+O código de teste não transmite suas intensões muito bem e possui vários trechos que podem ser refatorados.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md [hunk 27]
```diff
@@ -676,17 +670,19 @@ func assertContentType(t *testing.T, response *httptest.ResponseRecorder, want s
...
-Now that we have sorted out `PlayerServer` for now we can turn our attention to `InMemoryPlayerStore` because right now if we tried to demo this to the product owner `/league` will not work.
+Agora que nós resolvemos `PlayerServer`, por agora podemos mudar nossa atenção para `InMemoryPlayerStore` porque no momento se nós tentarmos demonstrá-lo para o gerente de produto, `/league` não vai funcionar.
+
+A forma mais rápida de nós termos alguma confiança é adicionar a nosso teste de integração, nós podemos bater no novo endpoint e checar se nós recebemos a resposta correta de `/league`.
 
-The quickest way for us to get some confidence is to add to our integration test, we can hit the new endpoint and check we get back the correct response from `/league`.
+## Escreva o teste primeiro
 
-## Write the test first
+Nós podemos usar `t.Run` para parar este teste um pouco e então reusar os helpers dos testes do nosso servidor - novamente mostrando a importancia de refatoração dos testes.
```
[the remark this answers is not in the document]
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

File: criando-uma-aplicacao/json.md (hunk 1 above)
Comment (Ronald-TR) [reply criando-uma-aplicacao/json.md]: Done!

**Comment by Ronald-TR** [comment]
Pessoal, desculpas não são suficientes, eu literalmente esqueci de resolver as revisões, deixei a bomba e sumi. Sorry!
Subi as correções solicitadas, requisitei novamente o review, espero não ter deixado nenhuma requisição de mudança passar batido!

Cheers!

**Comment by larien** [comment]
@Ronald-TR oxe, moço, todo mundo deu uma pausa esse ano, inclusive eu! Te preocupa não, o importante é que tamo de volta :) logo mais eu dou uma olhada na sua PR quando fechar as mais antigas
Obrigada novamente pelo trampo foda!

File: criando-uma-aplicacao/json/json.md [hunk 28]
```diff
@@ -487,28 +484,27 @@ func TestLeague(t *testing.T) {
...
 ```text
 ./server_test.go:33:3: too few values in struct initializer
 ./server_test.go:70:3: too few values in struct initializer
 ```
+## Escreva o minimo de código para que o teste rode e cheque as falhas na saída dele.
 
-## Write the minimal amount of code for the test to run and check the failing test output
-
-You'll need to update the other tests as we have a new field in `StubPlayerStore`; set it to nil for the other tests.
+Você vai precisar atualizar os outros testes, assim como nós temos um novo campo em `StubPlayerStore`; ponha-o como nulo para os outros testes.
```
Comment (devdrops) [inline criando-uma-aplicacao/json/json.md:495]: Talvez seja interessante manter a palavra `nil` ao invés de `null`, para manter a palavra reservada da linguagem.

Pode parecer purismo (o que não deixa de ser verdade :sweat_smile:), por isso pergunto: o que acha?

File: criando-uma-aplicacao/json/json.md [hunk 29]
```diff
@@ -722,15 +716,15 @@ func TestRecordingWinsAndRetrievingThem(t *testing.T) {
...
 ```text
 === RUN   TestRecordingWinsAndRetrievingThem/get_league
     --- FAIL: TestRecordingWinsAndRetrievingThem/get_league (0.00s)
         server_integration_test.go:35: got [] want [{Pepper 3}]
 ```
 
-## Write enough code to make it pass
+## Escreva código suficiente para fazê-lo passar
 
 `InMemoryPlayerStore` is returning `nil` when you call `GetLeague()` so we'll need to fix that.
```
Comment (devdrops) [inline criando-uma-aplicacao/json/json.md:729]: Sobrou este trecho em en_US :wink:

**Review by devdrops — CHANGES_REQUESTED** [review]
@Ronald-TR meus parabéns e muito obrigado pela tradução! :tada: 

Deixei algumas observações, e também peço que veja o seguinte ponto: alguns trechos de código exibem o nome `quii` ao invés de `larien`. Você pode corrigir isso usando o seguinte comando (requer sed):

```
sed -i 's/quii/larien/g' ./criando-uma-aplicacao/json/json.md
```

Se precisar de alguma ajuda, basta dar um salve ok? :wink:

**Comment by larien** [comment]
Oi! Passando para avisar que falta só esse capítulo para a tradução principal e vou colocar minhas mãos nele muitíssimo em breve. Agradeço desde já o esforço de vocês, viu? Cês são uns lindo <3

**Review by larien — APPROVED** [review]
Aprovando com as correções aplicadas e códigos traduzidos :)
